app/templates/components/ui-table/cell/cell-link.hbs

```diff
@@ -1,3 +1,16 @@
-<a href="{{this.record}}">
-  {{this.record}}
-</a>
+<div class="ui action input" style="width: 100%; height: 36px">
+  <input class="truncate" style="flex: 1 0 !important" type="text" value={{this.record}} placeholder={{this.record}} readonly>
+  <UiPopup @on="click" @content={{t "Link copied to clipboard"}} @position="bottom right">
+    <CopyButton @clipboardText={{this.record}} @class="ui grey icon button rounded-none m-0">
+      <i class="copy icon"></i>
+    </CopyButton>
+  </UiPopup>
+  <a 
+    id="public_url"
+    href="{{this.record}}" 
+    target="_blank" rel="noopener">
+    <div  class="ui blue icon button ml-auto rounded-l-none m-0">
+      <i class="share icon" ></i>
+    </div>
+  </a>
+</div>
```

Comment:
This component is being used a lot, make it a new component and reuse here and on video table